packages/hint-sri/package.json

```diff
@@ -2,7 +2,7 @@
   "ava": {
     "failFast": false,
     "files": [
-      "dist/tests/**/*.js"
+      "dist/tests/**/tests-https.js"
```

Comment:
Did you forget to restore this change?

Reply:
I did, but it's fixed now (caught it right after I mentioned everyone to look at the PR 🤦‍♂️)